src/main.c

```diff
@@ -164,6 +164,22 @@ long run_lagscope_sender(struct lagscope_test_client *client)
 	while (is_light_turned_on()) {
 		gettimeofday(&now, NULL);
 		send_time = now;
+
+		if (test->latte) {
+			test->iteration++;
```

Comment:
can you add comments here for the latte.exe behavior?
does latte.exe only care about the first 4 bytes?